README.md

```diff
@@ -0,0 +1,63 @@
+# GA4GH SDK
+
```

Comment:
Consider adding some shields/badges here. For now there may not be much to put yet, but at the very least, a badge for the license would be nice.

See here for examples/inspiration: https://github.com/elixir-cloud-aai/cookiecutter-python/blob/ea72128ea94f0693d2a940f3f1657554178b6b66/%7B%7B%20cookiecutter.project_name%20%7D%7D/README.md